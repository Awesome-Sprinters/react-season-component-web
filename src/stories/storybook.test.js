import initStoryshots from '@storybook/addon-storyshots';

initStoryshots({
  framework: 'react',
  test: ({ story }) => {
    const view = story.render();
    expect(view).toMatchSnapshot();
  },
});
