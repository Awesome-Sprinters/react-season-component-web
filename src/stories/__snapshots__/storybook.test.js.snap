// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Controller Auto 1`] = `
<Controller
  height={360}
  type="auto"
  width={500}
/>
`;

exports[`Storyshots Controller Spring 1`] = `
<Controller
  height={360}
  type="spring"
  width={500}
/>
`;

exports[`Storyshots Controller Winter 1`] = `
<Controller
  height={360}
  type="winter"
  width={500}
/>
`;

exports[`Storyshots SeasonPage Default 1`] = `
<HashRouter>
  <SeasonPage />
</HashRouter>
`;

exports[`Storyshots Sidebar Home 1`] = `
<HashRouter>
  <Sidebar
    currentPage="Home"
    type="home"
  />
</HashRouter>
`;

exports[`Storyshots TimePage Default 1`] = `
<HashRouter>
  <TimePage />
</HashRouter>
`;

exports[`Storyshots TypeChecker Default 1`] = `
<Controller
  height={360}
  onChangeHeight={[Function]}
  onChangeWidth={[Function]}
  type="auto"
  width={500}
>
  <TypeChecker
    handleType={[Function]}
  />
</Controller>
`;

exports[`Storyshots WeatherPage Default 1`] = `
<HashRouter>
  <WeatherPage />
</HashRouter>
`;
